web/src/modules/topic/components/Score/Score.tsx

```diff
@@ -34,10 +34,17 @@ export const Score = ({ arguableId, arguableType, score }: ScoreProps) => {
   const zoomRatio = useTopicZoom();
   const [selected, setSelected] = useState(false);
   const [hovering, setHovering] = useState(false);
-  const mainButtonRef = useRef(null);
+  const [refButtonHeight, setRefButtonHeight] = useState(0);
+  const mainButtonRef = useRef<HTMLButtonElement | null>(null);
 
-  const buttonLength = indicatorLength; //px
-  const buttonLengthScaled = buttonLength * zoomRatio; //px * zoom
+  useEffect(() => {
+    if (mainButtonRef.current?.clientHeight) {
+      setRefButtonHeight(mainButtonRef.current.clientHeight);
+    }
+  }, [mainButtonRef]);
+
```

Comment:
I'm using another state variable here so the client Height is never null.

I couldn't think of any better solution.

Ideas?